packages/icons/scripts/build.ts

```diff
@@ -51,7 +51,8 @@ const createManifestAndOptimizeIcons = (cdn: string, files: string[], config: Co
       } bytes (size: ${svgOptimizedSize} bytes)`
     );
 
-    if (svgOptimizedSize > 3000) {
+    // car icon size can't be more optimized it has 3.034KB
+    if (svgOptimizedSize > 3000 && svgRawName !== 'car') {
```

Comment:
Then the icon is not good enough. We can't ship a single icon that is as large as 2 component chunks.
You can see that at least the top right rounded corner isn't placed correctly.
<img width="780" alt="Bildschirm­foto 2023-03-02 um 08 27 24" src="https://user-images.githubusercontent.com/4380295/222360312-ea8fff5f-ea8d-4086-a1e6-752428a2623c.png">